tests/unitTests/test.h

```diff
@@ -951,7 +1105,9 @@ class TestHolzapfelOgden : public TestMaterialModel {
             Psi += a_f / (2.0 * b_f) * chi(I4_f, k) * (exp(b_f * pow(I4_f - 1.0, 2)) - 1.0);   // Fiber term
             Psi += a_s / (2.0 * b_s) * chi(I4_s, k) * (exp(b_s * pow(I4_s - 1.0, 2)) - 1.0);   // Sheet term
             Psi += a_fs / (2.0 * b_fs) * (exp(b_fs * pow(I8_fs, 2)) - 1.0);                   // Cross-fiber term
-            Psi += kappa / 4.0 * (pow(smTerms.J, 2) - 1.0 - 2.0 * log(smTerms.J));      // Simo-Taylor 91 volumetric penalty term
+            if (!ustruct) {
+                Psi += kappa / 4.0 * (pow(smTerms.J, 2) - 1.0 - 2.0 * log(smTerms.J));      // Add Simo-Taylor 91 volumetric penalty term
+            }
```

Comment:
It looks like the `!ustruct` part of the code is not executed. Maybe either add a test or delete?

Reply:
Yes thanks for pointing this out! I actually decided to remove the volumetric component from the HO model here, to be consistent with the other material model classes above. In doing so, I ended up removing this part of the code